src/site/_includes/macros/prerender.njk

```diff
@@ -3,7 +3,18 @@
 #}
 {% macro prerender(url) %}
 {% if url and not url.startsWith('http://') and not url.startsWith('https://') %}
-{% set apiScripts %}{"prerender":[{"source":"list","urls":["{{ url }}"]}]}{% endset %}
-<script type="speculationrules">{{ apiScripts | cspHash | safe }}</script>
+{% set speculationRules %}
+{
+  "prerender":[
+    {
+      "source":"list",
+      "urls":[
+        "{{ url }}"
+      ]
+    }
+  ]
+}
+{% endset %}
+<script type="speculationRules">{{ speculationrules | minifyJSON | cspHash | safe }}</script>
```

Comment:
The type is probably not case-sensitive, the variable name probably is...? As you are setting `speculation*R*ules` above.

Reply:
Oh good spot! Changed that right at the end of course!